.github/workflows/deploy.yml

```
@@ -1,4 +1,4 @@
-name: Build and Deploy Erimodo bot
+name: Deploy Erimodo bot
```

```suggestion
name: Toto
```

Je ne pense pas que ce code est bon !